### DISCUSSION TITLE
Duplicate items in meal planner (i.e. leftovers)

### DISCUSSION THREAD

**Discussion by davidjf1** [mealie-recipes/mealie#4139] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

An easy way to duplicate a recipe within the meal planner, for example, dinner on Monday can easily be added to Tuesday lunch. Something like a leftovers feature. 

### Please provide a concise description of the feature that would resolve your issue.

A duplicate option within the list within the ‘Edit’ section on meal planner, or being able to assign a meal to more than one day at a time. 

### Please consider and list out some caveats or tradeoffs made in your design decision

Slightly longer context menu

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by saraheary19** [comment]
This feature request seems similar to this feature request: Recurring meals in meal plan #4525 [https://github.com/mealie-recipes/mealie/discussions/4525](url)

**Reply by davidjf1** [reply]
I was thinking more of a ‘leftovers’ feature. Something that would make it really easy to assign the same meal to lunch or dinner the next day for example, or at least something that allows you to do it from the Meal Plan Editor instead of having to open the recipe up and repeat it again. 

Even just being able to add a recipe to multiple days in the meal planner would work.
